src/components/features/ctf-components/ctf-expert-advice/ctf-expert-advice.tsx

```diff
@@ -16,44 +16,29 @@ export default function CtfExpertAdvice({title, subtitle, buttonText1, buttonTex
             <h2 className='text-white font-bold pt-4 pb-4'>{title}</h2>
             <p className='text-white pb-4 text-base'>{subtitle}</p>
             <div className='flex self-stretch flex-col gap-y-12 mt-4 mb-8'>
-              <button
-                className="py-3.5 pl-5 pr-12 bg-white text-gray-600 text-base text-start flex flex-row justify-between">
-                {buttonText1}
-                <div className="h-full flex items-center">
-                  <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5}
-                       stroke="currentColor" className="w-6 h-6">
-                    <path strokeLinecap="round" strokeLinejoin="round"
-                          d="m12.75 15 3-3m0 0-3-3m3 3h-7.5M21 12a9 9 0 1 1-18 0 9 9 0 0 1 18 0Z" />
-                  </svg>
-                </div>
-              </button>
-              <button
-                className="py-3.5 pl-5 pr-12 bg-white text-gray-600 text-base text-start flex flex-row justify-between">
-                {buttonText2}
-                <div className="h-full flex items-center">
-                  <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5}
-                       stroke="currentColor" className="w-6 h-6">
-                    <path strokeLinecap="round" strokeLinejoin="round"
-                          d="m12.75 15 3-3m0 0-3-3m3 3h-7.5M21 12a9 9 0 1 1-18 0 9 9 0 0 1 18 0Z" />
-                  </svg>
-                </div>
-              </button>
-              <button
-                className="py-3.5 pl-5 pr-12 bg-white text-gray-600 text-base text-start flex flex-row justify-between">
-                The content is at the heart of the UX, but watch out for silos!
-                <div className="h-full flex items-center">
-                  <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5}
-                       stroke="currentColor" className="w-6 h-6">
-                    <path strokeLinecap="round" strokeLinejoin="round"
-                          d="m12.75 15 3-3m0 0-3-3m3 3h-7.5M21 12a9 9 0 1 1-18 0 9 9 0 0 1 18 0Z" />
-                  </svg>
-                </div>
-              </button>
+              {
+                advices && advices.items.map((item) => (
+                  <div key={item.title}>
```

Comment:
Wouldn't it make more sense to make a singlar advice component then iterate it with the data?